config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -550,6 +550,24 @@ presubmits:
         command:
         - ./test/presubmit-backend-visualization.sh
 
+  - name: kubeflow-pipeline-multiuser-test
+    cluster: kubeflow
+    always_run: false
+    decorate: true
+    skip_report: true
```

Comment:
@chensun I think you need to change this to false to let it show in PR status.

Reply:
I see. Thank you! I'll send a PR shortly.

Other than this, there're still some issues I haven't had a chance to take a closer look.